jenkinsfile.groovy

```diff
@@ -462,6 +491,40 @@ pipeline {
       } //parallel
     } //stage deploy
 
+    stage ("Load data to DB") {
+      when  { expression { params.action == 'apply' }}
+      agent { node { label "${jenkinsCtrlNodeLabel} && ${account}" } }
```

Comment:
not sure if `jenkinsCtrlNodeLabel` is appropriate for tasks involving `npm install`, I'd try using `jenkinsBuildLabel` node